voila/handler.py

```diff
@@ -133,15 +135,14 @@ def _jinja_cell_generator(self, nb, kernel_id):
         """Generator that will execute a single notebook cell at a time"""
         km = self.kernel_manager.get_kernel(kernel_id)
 
-        all_cells = list(nb.cells)  # copy the cells, since we will modify in place
-        for cell in all_cells:
-            # we execute one cell at a time
-            nb.cells = [cell]  # reuse the same notebook
-            result = executenb(nb, km=km, cwd=self.cwd, config=self.traitlet_config)
-            cell = result.cells[0]  # keep a reference to the executed cell
-            nb.cells = all_cells  # restore notebook in case we access it from the template
-            # we don't filter empty cells, since we do not know how many empty code cells we will have
-            yield cell
+        nb, resources = ClearOutputPreprocessor().preprocess(nb, {'metadata': {'path': self.cwd}})
+        ep = VoilaExecutePreprocessor(config=self.traitlet_config)
```

Comment:
We are missing https://github.com/voila-dashboards/voila/commit/c21b216e227656521da419a82815f001e510b4b0  (see https://github.com/voila-dashboards/voila/blame/master/voila/execute.py#L159)

Reply:
Mmh... Isn't it line `138`?